src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Input, Error } from './style';
+
+const TextField = (values = '') => {
+  const { value, disabled, error } = values;
+  return (
+    <>
+      <Input type="text" value={value} disabled={disabled} />
+      {error ? <Error>{error}</Error> : <br />}
```

Comment:
instead use of br in ternaray operator use null or blank string like - 
error ? <Error>{error}</Error> : ''

Reply:
Done